test/e2e/manifests/ops/operations/basic/setup/functions.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# We intentionally use v1beta1 here, to make sure it still works.
+apiVersion: pkg.crossplane.io/v1beta1
+kind: Function
+metadata:
+  name: function-dummy-ops-basic
+spec:
+  # NOTE(negz): This is currently manually pushed. See README.md at
+  # https://github.com/crossplane-contrib/function-dummy.
+  # NOTE(negz): This version of the function uses meta.pkg.crossplane.io/v1beta1
+  # and is built with an old SDK that only serves v1beta1 RPCs. This is
+  # intentional. We want to make sure Crossplane is backward compatible with
+  # older v1beta1 functions.
+  package: xpkg.upbound.io/crossplane-contrib/function-dummy:v0.2.1
```

Comment:
I need to update this to use a build of https://github.com/crossplane-contrib/function-dummy/pull/42. So please prioritize reviewing that, and the function-sdk-go update it depends on in turn. 😄 

The ops E2E is currently failing because this function doesn't have the required `operation` capability.

Once I've updated it I can also update the E2E to test returning output from a function.

Reply:
Done!